gke/private_kubernetes/variables.tf

```diff
@@ -128,4 +128,10 @@ variable "private_vms" {
 
 variable "privileged_bastion" {
   type = bool
-}
\ No newline at end of file
+}
+
+variable "preemptible_nodes" {
```

Comment:
Can the word "nomad" be added to the name, like is done for k8s

Reply:
This adds preemptible nodes to the kubernetes cluster. There is no prefix here because this is inside the kubernetes module but it is called `preemptible_k8s_nodes` in the top level.

Reply:
of course, of course 👍